### ISSUE TITLE
How to make the autocomplete dropdown display whole words?

### ISSUE THREAD

**Issue by HebaruSan** [atom/autocomplete-plus#991]
### Description

The autocomplete dropdown is obscuring the choices it presents. Try to guess which word any of these options represent:

![image](https://user-images.githubusercontent.com/1559108/45163109-28698100-b1b5-11e8-9ced-78ad7663bdc4.png)

### Steps to Reproduce

1. Open any file with autocomplete enabled
2. Start typing something
3. The autocomplete dropdown appears

**Expected behavior:**

The dropdown should show the words that can be selected. This was the case at some time in the recent past.

**Actual behavior:**

Only the letters to be added are shown; the ones the user has already typed are hidden. This makes the dropdown far more confusing to use; in the best case, you can guess which words are being presented with some significant concentration, but in the worst case it's a total mystery, making the dropdown unusable. See screenshot above.

**Reproduces how often:**

100%

### Versions

```
Atom    : 1.29.0
Electron: 2.0.5
Chrome  : 61.0.3163.100
Node    : 8.9.3
Ubuntu  : 18.04
```

**Comment by Aerijo** [comment]
@HebaruSan I can't reproduce on Atom 1.29.0

I don't recognise those completions either; are they being provided by a community package? Could you try Atom in safe mode to see if the issue persists? If it doesn't, then you should raise this issue on the repo of that community package.

It may also be that excessively long descriptions are causing it. But that seems less likely, because there is still some space between the lower completions and their descriptions.

**Comment by HebaruSan** [comment]
No, no community package is involved. The same issue occurs when editing code, so I don't think it's on the completion-providing side, since multiple providers would have to have the same bug:

![image](https://user-images.githubusercontent.com/1559108/45164276-c1999700-b1b7-11e8-8604-6436b414eb84.png)

![image](https://user-images.githubusercontent.com/1559108/45164438-0f160400-b1b8-11e8-8736-6d09e6e3d483.png)

**Comment by Aerijo** [comment]
@HebaruSan Can you please confirm this was done in safe mode (open Atom with `atom --safe`)

**Comment by HebaruSan** [comment]
Yes this happens in safe mode.

**Comment by HebaruSan** [comment]
Now the find pane is doing it, showing me a blank underline instead of the matching string:

![image](https://user-images.githubusercontent.com/1559108/56610413-fc303280-65d4-11e9-9b55-d6e840e295bc.png)

Do these places maybe use different fonts? Maybe those fonts are invalid?

**Comment by HebaruSan** [comment]
Here are my Editor font settings, now on Ubuntu 19.04 but originally on 18.04 or 18.10:

![image](https://user-images.githubusercontent.com/1559108/56610847-13235480-65d6-11e9-8f31-92aa2c92e743.png)

**Comment by HebaruSan** [comment]
The font does seem to be a factor. If I toggle the `font-family` attribute of `div.results-view.focusable-panel` in the dev tools, the text comes back in the find pane. Before:

![image](https://user-images.githubusercontent.com/1559108/56611626-d5273000-65d7-11e9-94cc-5c3062158d2b.png)

After:

![image](https://user-images.githubusercontent.com/1559108/56611633-da847a80-65d7-11e9-8dbf-7a142d374a69.png)

I've tried to check the same thing for autocomplete, but the dropdown disappears when I click in the dev tools. Any way to avoid that?

**Comment by HebaruSan** [comment]
For autocomplete, the font is again determinative, but in this case it's the toggling of merely `font-weight:bold` in the `character-match` class that makes the text appear and disappear. Before:

![image](https://user-images.githubusercontent.com/1559108/56620711-7a013780-65ef-11e9-8dd5-7bc238df450f.png)

After:

![image](https://user-images.githubusercontent.com/1559108/56620718-7cfc2800-65ef-11e9-99ae-32d559d2a0f5.png)

**Comment by HebaruSan** [comment]
I assume there's something wrong with "Source Code Variable". I've switched my font setting to Inconsolata, and both autocomplete and find now work fine with all text displayed as expected.

Leaving open in case an Atom team member wants to look into it further; maybe a code change could make "Source Code Variable" work here as well.
